source/docs/articles/sites/code/applying-upstream-updates.md

```diff
@@ -60,15 +60,24 @@ This does not solve all problems that may arise, but it should take care of most
 
 In the event the "Auto-Resolve Conflicts" option fails, the next step is to manually pull your changes in using Git, resolve the conflicts, and then push the update up to your Pantheon site.
 
+### Resolve Conflicts Locally With Drupal 8
+From within an up-to-date Git clone on your local machine:
+```bash
+git pull git://github.com/pantheon-systems/drops-8.git master
+# resolve conflicts
+git push origin master
+```
+Add the -Xtheirs flag if you want to accept all changes.

```

Comment:
Can we consolidate these H3's into one command sequence, preceded by a list of or dropdown for selecting the git url they need. The `-Xtheirs` flag should be in backticks. If we don't consolidate, it should be mentioned on all subsections.